submissions/management/commands/update_submissions_uuids.py

```diff
@@ -56,12 +57,12 @@ def handle(self, *args, **options):
         total_len = Submission.objects.count()
         log.info("Beginning uuid update, {} rows exist in total".format(total_len))
 
-        current = options['start'];
+        current = options['start']
         while current < total_len:
             end_chunk = current + options['chunk'] if total_len - options['chunk'] >= current else total_len
             log.info("Updating entries in range [{}, {})".format(current, end_chunk))
             with transaction.atomic():
-                for submission in Submission.objects.filter(id__gte=current, id__lt=end_chunk).iterator():
+                for submission in Submission.objects.all()[current:end_chunk].iterator():
```

Comment:
Does all actually do the select(*) ?  I assume the slice is happening in python space?  Maybe the ORM is actually clever here.